### PR TITLE
Add MONGODB_TEXT_SEARCH_ENABLED flag

### PR DESCRIPTION
**Pull request by mfojtik**
See: https://trello.com/c/XGCb85EF/908-allow-enabling-text-search-with-mongodb-image

### CODE REVIEW

**Comment by mfojtik** [comment]
[test]

@bparees PTAL

File: 2.6/README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ The following environment variables influence the MongoDB configuration file. Th
 |  `MONGODB_NOPREALLOC` | Disable data file preallocation.                                          |  true
 |  `MONGODB_SMALLFILES` | Set MongoDB to use a smaller default data file size.                      |  true
 |  `MONGODB_QUIET`      | Runs MongoDB in a quiet mode that attempts to limit the amount of output. |  true
+|  `MONGODB_TEXT_SEARCH_ENABLED` | Enables the [text search](https://docs.mongodb.org/v2.4/core/index-text/#text-search-text-command) feature.  |  false
```
Comment (php-coder) [inline 2.6/README.md]: Maybe it's better to use link for 2.6 here?

Comment (mfojtik) [inline 2.6/README.md]: they are the same :-) but sure.

File: 2.4/root/usr/share/container-scripts/mongodb/mongodb.conf.template [hunk 2]
```diff
@@ -19,3 +19,8 @@ quiet = ${MONGODB_QUIET}
 
 # Disable the HTTP interface (Defaults to localhost:28017).
 nohttpinterface = true
+
+# If the flag is not enabled, you cannot create new text indexes, and you cannot perform text searches. 
+# Since it is not recommended to have this on in production, the default is
+# "false".
```
Comment (php-coder) [inline 2.4/root/usr/share/container-scripts/mongodb/mongodb.conf.template]: Maybe join together last 2 lines and make comment more compact?

File: 2.4/root/usr/bin/run-mongod [hunk 3]
```diff
@@ -15,6 +15,7 @@ export MONGODB_DATADIR=/var/lib/mongodb/data
 export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
 export MONGODB_SMALLFILES=${MONGODB_SMALLFILES:-true}
 export MONGODB_QUIET=${MONGODB_QUIET:-true}
+export MONGODB_TEXT_SEARCH_ENABLE=${MONGODB_TEXT_SEARCH_ENABLE:-false}
```
Comment (bparees) [inline 2.4/root/usr/bin/run-mongod]: is it ENABLED or ENABLE?  doc says ENABLED, this says ENABLE
(should be ENABLED imho)

**Comment by bparees** [comment]
one significant nit, otherwise lgtm.

**Comment by mfojtik** [comment]
@bparees good catch, fixed.

**Pull request merged**